### PR TITLE
Added support for specifying tool and is_open using env variables

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: migrate_issues.rb [hunk 1]
```diff
@@ -190,9 +190,13 @@ def hendl_pr(original)
 require './tools'
 names = @tools.reject { |tool| tool == "fastlane" } # we don't want to import issues from our own repo
 destination = "fastlane/playground" # TODO: Should be fastlane
+names = [ENV["TOOL"]] if ENV["TOOL"]
+open_only = true
+open_only = false if ENV["ALL"]
```
Comment (mfurtak) [inline migrate_issues.rb]: Replace two lines with `open_only = !ENV["ALL"]`?

File: migrate_issues.rb [hunk 2]
```diff
@@ -190,9 +190,13 @@ def hendl_pr(original)
 require './tools'
 names = @tools.reject { |tool| tool == "fastlane" } # we don't want to import issues from our own repo
 destination = "fastlane/playground" # TODO: Should be fastlane
+names = [ENV["TOOL"]] if ENV["TOOL"]
```
Comment (mfurtak) [inline migrate_issues.rb]: names portion could be?

```ruby
names = ENV["TOOL"] ? [ENV["TOOL"]] : @tools.delete("fastlane") # we don't want to import issues from our own repo
```

Comment (segiddins) [inline migrate_issues.rb]: golfing it a bit
`Array(ENV["TOOL"] || @tools.delete("fastlane"))`

File: migrate_issues.rb (hunk 1 above)
Comment (KrauseFx) [inline migrate_issues.rb]: Sounds good

File: migrate_issues.rb (hunk 2 above)
Comment (KrauseFx) [inline migrate_issues.rb]: Thanks @segiddins :+1:

**Comment by KrauseFx** [comment]
@mfurtak How is that :+1:

**Comment by mfurtak** [comment]
:cool: :ship: it

**Pull request merged**